src/fig/2.0.0.ts

```diff
@@ -0,0 +1,4146 @@
+        {
+          name: "autocomplete-window",
+          description: "Toggle/set autocomplete window debug mode",
+          options: [
+            {
+              name: ["-h", "--help"],
+              description: "Print help information",
+            },
+          ],
+          args: {
+            name: "mode",
+            isOptional: true,
+            suggestions: ["on", "off"],
+          },
+        },
+        {
+          name: "logs",
+          description: "Show fig debug logs",
+          options: [
+            {
+              name: "--level",
+              isRepeatable: true,
+              args: {
+                name: "level",
+                isOptional: true,
+              },
+            },
+            {
+              name: ["-h", "--help"],
+              description: "Print help information",
+            },
+          ],
+          args: {
+            name: "files",
+            isVariadic: true,
+            isOptional: true,
+            generators: subsystemsGenerator,
+          },
+        },
+        {
+          name: "prompt-accessibility",
+          description: "Prompt accessibility",
+          options: [
+            {
+              name: ["-h", "--help"],
+              description: "Print help information",
+            },
+          ],
+        },
+        {
+          name: "sample",
+          description: "Sample fig process",
+          options: [
+            {
+              name: ["-h", "--help"],
+              description: "Print help information",
+            },
+          ],
+        },
+        {
+          name: "unix-socket",
+          description: "Debug fig unix sockets",
+          options: [
+            {
+              name: ["-h", "--help"],
+              description: "Print help information",
+            },
+          ],
+        },
+        {
+          name: "verify-codesign",
+          description: "Debug fig codesign verification",
+          options: [
+            {
+              name: ["-h", "--help"],
+              description: "Print help information",
+            },
+          ],
+        },
+        {
+          name: "accessibility",
+          description: "Accessibility",
+          options: [
+            {
+              name: ["-h", "--help"],
+              description: "Print help information",
+            },
+          ],
+          args: {
+            name: "action",
+            isOptional: true,
+            suggestions: ["refresh", "reset", "prompt", "open", "status"],
+          },
+        },
+        {
+          name: "key-tester",
+          description: "Key Tester",
+          options: [
+            {
+              name: ["-h", "--help"],
+              description: "Print help information",
+            },
+          ],
+        },
+        {
+          name: "diagnostics",
+          description: "Watches diagnostics",
+          options: [
+            {
+              name: "--rate",
+              isRepeatable: true,
```

Comment:
This does not sound right to me, is this option really repeatable?
Same for a bunch of other options like the ones from the `query-index` subcommand.
```suggestion
              isRepeatable: true,
```

Reply:
Yeah, that doesnt look right, hmm, an issue with the clap generator perhaps

Reply:
Let's fix this in an update to the spec - imo for now we should just land this PR. If it's an error in the clap generator, we'll remove it later, if it's in the CLI itself then we'll fix it with semver

Reply:
Great! @SeparateRecords if everything looks good to you feel free to merge.